code/implementation/docker/machine/src/main/resources/schema/base/rackspaceConfig.json.d/rackspace.json

```diff
@@ -0,0 +1,42 @@
+{
+    "resourceFields": {
+        "username": {
+            "type": "string",
+            "nullable": false,
+            "required": true
+        },
+        "apiKey": {
+            "type": "string",
+            "nullable": false,
+            "required": true
+        },
+        "region": {
+            "type": "string",
+            "nullable": true
+        },
+        "endpointType": {
+            "type": "string",
+            "nullable": true
+        },
+        "imageId": {
+            "type": "string",
+            "nullable": true
+        },
+        "flavorId": {
+            "type": "string",
+            "nullable": true
+        },
+        "sshUser": {
+            "type": "string",
+            "nullable": true
+        },
+        "sshPort": {
+            "type": "string",
+            "nullable": true
+        },
+        "dockerInstall": {
```

Comment:
@cjellick I have tested that one specifically. The same question came up while writing this and I tested it.